v2/spanner-common/pom.xml

```diff
@@ -52,6 +52,73 @@
                         <phase>none</phase>
                     </execution>
                 </executions>
+            </plugin>
+		    <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>${jacoco.version}</version>
+                <configuration>
+                    <excludes>
+                        <!-- Excluding exception classes -->
+                        <exclude>**/*Exception.*</exclude>
+                        <!-- Excluding auto-generated classes. -->
+                        <exclude>**/*AutoValue_*</exclude>
+                    </excludes>
+                    <rules>
```

Comment:
Line 67 to 81 is not needed. These inclusion and exclusions correspond to Spanner Import/Export templates.